packages/provider/src/mobile.ts

```diff
@@ -11,3 +11,6 @@ initInpageProvider({
     version: 'replace_version',
   },
 });
+// note: this is required, or you'll sometimes get silent failures
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+true;
```

Comment:
source?

Reply:
That is so strange - kind of annoying the docs have that comment but no real explanation on what's going on.